resourceDefinition.json

```diff
@@ -1037,6 +1037,50 @@
         "lowercase": false,
         "regex": "\"[^0-9A-Za-z<>*%:.?\\\\+\\\\/]\""
     },
+    {
+        "name": "azurerm_data_protection_backup_vault",
+        "min_length": 2,
+        "max_length": 50,
+        "validation_regex": "\"^[a-zA-Z][a-zA-Z0-9-]{1,49}$\"",
+        "scope": "resourceGroup",
+        "slug": "dpbv",
+        "dashes": true,
+        "lowercase": false,
+        "regex": "`[^a-zA-Z0-9-]`"
+    },
+    {
+        "name": "azurerm_data_protection_backup_policy_disk",
+        "min_length": 3,
+        "max_length": 150,
+        "validation_regex": "\"^[a-zA-Z][a-zA-Z0-9\\\\-]{1,148}[a-zA-Z0-9]$\"",
+        "scope": "resourceGroup",
+        "slug": "dpbpd",
+        "dashes": true,
+        "lowercase": false,
+        "regex": "`[^a-zA-Z0-9-]`"
+    },
+    {
+        "name": "azurerm_data_protection_backup_policy_blob_storage",
+        "min_length": 3,
+        "max_length": 150,
+        "validation_regex": "\"^[a-zA-Z][a-zA-Z0-9\\\\-]{1,148}[a-zA-Z0-9]$\"",
+        "scope": "resourceGroup",
+        "slug": "dpbpb",

```

Comment:
2 items are sharing the same slug this would cause conflicts...

Reply:
@sschne have you been able to review @Nepomuceno comment?

Reply:
Commenting on behalf of @sschne : no conflict with `dpbpp`